sdk/helper/pluginutil/run_config_test.go

```diff
@@ -322,7 +322,7 @@ func TestMakeConfig(t *testing.T) {
 			responseWrapInfoTimes: 0,
 
 			mlockEnabled:      false,
-			mlockEnabledTimes: 1,
```

Comment:
@tomhjp I'm not confident about the changes in this file, would appreciate if you could take a look to make sure that this is the intended behavior

Reply:
Yeah this is correct - it checks once for the env var, and once for setting the IPC_LOCK config option in `plugincontainer`.

Reply:
And `GRPCBrokerMultiplex` recently became the default when using a container image.